src/Components/Form_register/Form_Reg.css

```
@@ -0,0 +1,14 @@
+
+form {
+    text-align: left !important;
+    margin: 0 auto;
+    width: 400px;
+}
+
+table th {
+    text-align: center;
+}
+
+table th, td{
```

```suggestion
table th,
table td {
```
Mỗi 1 selector hãy viết trên 1 dòng
Chú ý các dấu cách trước dấu `{`